vendor/bat-native-ledger/src/bat/ledger/internal/database/database.h

```diff
@@ -246,15 +246,15 @@ class Database {
   /**
    * PROMOTION
    */
-  void SavePromotion(
+  virtual void SavePromotion(
```

Comment:
For testing purposes?

Reply:
yes, so that we can mock functions in unit tests